### DISCUSSION TITLE
Fn+PAUSE(F14) or Fn+SCRLK(F15) keyboard shortcuts not working on some configurations?

### DISCUSSION THREAD

**Discussion by bigplayer-ai** [MonitorControl/MonitorControl#1638] [General]
# Issue: External Keyboard Brightness Control Not Working on All Monitors

## Pre-Submission Checklist

- [X] Checked for existing issues
- [X] Reviewed the wiki
- [X] Updated to the latest MonitorControl version

## Bug Description

Using the external keyboard's Control + Pause shortcut fails to adjust the brightness on all connected monitors.

**Shortcut Keys:**
- Increase Brightness: Control/Fn + PAUSE (F14)
- Decrease Brightness: Control/Fn + SCRLK (F15)

## Reproduction Steps

1. Attempt to adjust brightness with the external keyboard using the above shortcuts.
2. Only the MacBook Pro's native screen responds; external monitors remain unaffected.

## Expected Outcome

All monitors should respond to brightness adjustment shortcuts.

## System Information

- **macOS**: Sonoma 14.4
- **Mac Model**: MacBook Pro 16" 2021 M1 Pro
- **MonitorControl Version**: 4.2.0
- **Monitors**: One external, one internal
- **Apple Silicon**: Yes

**Comment by mdevils** [comment]
Same problem, but with standard brightness control buttons and an external keyboard. It used to work fine before with external keyboards.

**Comment by LukasGirsch** [comment]
I have the same issue. Adjusting the brightness with the slider on screen works on all monitors however when using the external keyboard only the main monitors adjusts its brightness.

**Comment by sihil** [comment]
Experiencing the same issue on macOS 14.5. Think it started with a recent upgrade to 14.4 but prior to that I was on 14.1 where the external keyboard would control the screen where the mouse pointer was. The integrated keyboard still does this but the external keyboard only controls the integrated screen.

**Comment by jonnyyu** [comment]
same issue here.

**Comment by jussikuosa** [comment]
+1 Using 4.2.0 build 7048 on Intel mbp with Sonoma 14.7 and external Apple (small) keyboard. Laptop brightness touch bar control works for both monitors and external only controls the one where mouse is.

**Comment by waydabber** [comment]
You probably need to fix accessibility permissions.

**Comment by waydabber** [comment]
I am not sure whether this is still an issue with the current version. Anyway, with a PC keyboard (from the description it seems like this is involved) I simply recommend using a custom keyboard shortcut.
